app/models/base.py

```diff
@@ -40,3 +45,54 @@ class Config:
         match their type annotations"""
 
         validate_assignment = True
+
+
+class BaseResponse(TableModelMixin):
+    pass
+
+
+class BaseRequest(BaseModel):
+    class Meta:
+        related_fields = {}
+        model = None
+
+    def get_model(self):
+        if not self.Meta.model:
+            raise NotImplementedError(
+                "Either set the Meta.model property or override the get_model() method."
+            )
+        return self.Meta.model
+
+    def get_related_fields(self):
+        return self.Meta.related_fields
```

Comment:
Likewise here, could you provide type hints and potential turn this into a property, as it is just returning a static dict.

For example
```python
class Meta:
  related_fields: dict[str] = {}
  model: BaseModel | None = None

@property 
def related_fields(self) -> dict[str]
   return self.Meta.related_fields
```

Then you can access the requests related fields using the Request.related_fields syntax.

Reply:
Updated with type hinting and renaming to `related_fields`